packages/sdk/src/lib/dataProtectorCore/grantAccess.ts

```diff
@@ -98,7 +99,9 @@ export const grantAccess = async ({
     throw new WorkflowError({
       message: grantAccessErrorMessage,
       errorCause: Error(
-        `An access has been already granted to the user: ${vAuthorizedUser} with the app: ${vAuthorizedApp}`
+        `An access has been already granted to the user: ${
+          vAuthorizedUser || NULL_ADDRESS
+        } with the app: ${vAuthorizedApp}`
```

Comment:
Improve error message, authorized user can be undefined.

Previously:

```
An access has been already granted to the user: undefined with the app: 0x7a8f4c23ef61dd295b683409fe15ad76bc92c14e
```

After:

```
An access has been already granted to the user: 0x0000000000000000000000000000000000000000 with the app: 0x7a8f4c23ef61dd295b683409fe15ad76bc92c14e
```